client/my-sites/plugins/main.jsx

```diff
@@ -37,6 +41,7 @@ import {
 	getSelectedSiteId,
 	getSelectedSiteSlug,
 } from 'calypso/state/ui/selectors';
+import siteHasFeature from '../../state/selectors/site-has-feature';
```

Comment:
It looks like the other imports from state use an absolute path. Let's update this and the others to `'calypso/state/selectors/site-has-feature'`?